tasks/wait-for-image-task.yaml

```
@@ -9,6 +9,10 @@ spec:
     description: Image reference.
     type: string
   results:
+  - name: GIT_REF
+    description: Git commit for the image's source code from `vcs-ref` image label.
+  - name: GIT_REPO
+    description: Git repository URL for the image's source code from `source-location` image label.
   - name: IMAGE_DIGEST
     description: Image digest in the format `sha256:abcdef0123`.
```

[nit] I suggest to reorder params to match how they come out in `infos`.
```suggestion
  - name: IMAGE_DIGEST
    description: Image digest in the format `sha256:abcdef0123`.
  - name: GIT_REF
    description: Git commit for the image's source code from `vcs-ref` image label.
  - name: GIT_REPO
    description: Git repository URL for the image's source code from `source-location` image label.
```